components/Cloudinary.js

```diff
@@ -20,24 +18,53 @@ export default function CloudinaryUpload() {
             'upload_preset',
             process.env.NEXT_PUBLIC_UPLOAD_PRESET
           );
+
+          if (audioValue === '') {
+            toast('Your haven`t selected a file!', {
+              hideProgressBar: true,
+              autoClose: 1000,
+              type: 'info',
+              position: 'top-center',
+            });
+            return false;
+          } else {
+            toast('Good job!', {
+              hideProgressBar: true,
+              autoClose: 500,
+              type: 'success',
+              position: 'top-center',
+            });
+          }
+
           const response = await fetch(
-            `https://api.cloudinary.com/v1_1/${process.env.NEXT_PUBLIC_CLOUDNAME}/upload`
-            // {
-            //   method: 'POST',
-            //   body: formData,
-            // }
+            `https://api.cloudinary.com/v1_1/${process.env.NEXT_PUBLIC_CLOUDNAME}/upload`,
+            {
+              method: 'POST',
+              body: formData,
+            }
           );
           const json = await response.json();
           console.log(json);
```

Comment:
```suggestion

```

Reply:
Actually this one's kinda helpful/necessary in the Cloudinary upload process.